### ISSUE TITLE
Progpowz algo dont work

### ISSUE THREAD

**Issue by alvarezdzudi** [UnamSanctam/UnamWebPanel#401]
in builder Progpowz  algorithm doesn't work.

**Comment by UnamSanctam** [comment]
It does work it's just that all the old big pools have closed down and only the small ones remain, and I think they require some sort of special connection parameters or something.

**Comment by alvarezdzudi** [comment]
zano.luckypool.io pool progpowz

**Comment by UnamSanctam** [comment]
Yes that pool does not seem to work, the other pools that used to exist worked just fine but that pool does not seem to work, I assume it does something special that would need to be implemented in the miner. Firo should work though, but I would probably recommend something else like Neurai https://whattomine.com/.
